templates/concierge.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Concierge Miis</title>
+        <link rel="stylesheet" href="https://yuzu.api.wiilink24.com/css0.css"/>
```

Comment:
Please keep this to `templates/common.css` so that changes locally will reflect.

Reply:
That I can do, I'll read through and respond to everything :) But yeah, the PHP link, something IS defintely missing, and I do kinda need your help with that.